sympy/integrals/tests/test_risch.py

```diff
@@ -226,17 +227,17 @@ def test_residue_reduce():
     assert residue_reduce(a, d, DE, z, invert=True) == \
         ([(Poly(z**2 - Rational(1, 4), z, domain='ZZ(x)'), Poly(t + 2*x*z, t))], False)
     assert residue_reduce(Poly(-2/x, t), Poly(t**2 - 1, t,), DE, z, invert=False) == \
-        ([(Poly(z**2 - 1, z, domain='QQ'), Poly(-2*z*t/x - 2/x, t, domain='ZZ(z,x)'))], True)
+        ([(Poly(z**2 - 1, z, domain='ZZ(x)'), Poly(-z*t/x - 1/x, t, domain='ZZ(z,x)'))], True)
```

Comment:
Here a factor of `2` is removed. I am not sure if that matters or not in context...